### PR TITLE
New recipe: weightedstats (0.3)

### PR DESCRIPTION
**Pull request by HadrienG**

### CODE REVIEW

**Comment by druvus** [comment]
@conda-forge/help-python could someone please review?

File: recipes/weightedstats/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - weightedstats
+
+about:
+  home: https://github.com/tinybike/weightedstats
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: Mean, weighted mean, median, weighted median
+  doc_url: 
+  dev_url:
```
Comment (beckermr) [inline recipes/weightedstats/meta.yaml]: These should be filled in or removed.

Comment (HadrienG) [reply recipes/weightedstats/meta.yaml]: changed in 7d020f

**Comment by HadrienG** [comment]
@conda-forge/help-python could someone re-review? Changes not marked at resolved for some reason.

**Pull request merged**